distributed/worker.py

```diff
@@ -3152,7 +3196,6 @@ async def find_missing(self) -> None:
                 self.periodic_callbacks[
                     "find-missing"
                 ].callback_time = self.periodic_callbacks["heartbeat"].callback_time
-                self.ensure_communicating()
```

Comment:
This is no longer necessary as _ensure_communicating is now in all transitions to fetch.